vision/dcgan_mnist/dcgan_mnist.jl

```diff
@@ -30,12 +31,18 @@ function create_output_image(gen, fixed_noise, hparams)
     return image_array
 end
 
+  
+# weight initialization
+function random_normal(shape...)
+    return map(Float32,rand(Normal(0,0.02),shape...))
+end
+
 function Discriminator()
     return Chain(
-            Conv((4, 4), 1 => 64; stride = 2, pad = 1),
+            Conv((4, 4), 1 => 64; stride = 2, pad = 1,init=random_normal),
```

Comment:
Match the formatting spacing here and elsewhere.

```suggestion
            Conv((4, 4), 1 => 64; stride = 2, pad = 1, init = random_normal),
```